vectordraw.js

```diff
@@ -329,7 +393,7 @@ VectorDraw.prototype.canCreateVectorOnTopOf = function(el) {
     if (el instanceof JXG.Point) {
         var vector = this.getVectorForObject(el);
         if (!vector) {
-            return true;
+            return !(el instanceof JXG.Point) || el.getProperty('fixed');
```

Comment:
@smarnach we're in `el instanceof JXG.Point` branch, so first part of condition is always false